src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTest/UnitTestsControl.cs

```diff
@@ -217,6 +220,11 @@ void Update()
 					}
 				}
 
+				if (console is { })
+				{
+					testResultBlock.Inlines.Add(new Run { Text = "\nOUT>" + console, Foreground = new SolidColorBrush(Colors.Gray) });
```

Comment:
We may want to disable this by default, this can be very noisy.

Reply:
It is disabled by default!!  You need to press the `<ToggleButton />` in the page to activate it.